spec/index.html

```diff
@@ -508,6 +508,15 @@ <h3>Overview</h3>
       hexadecimal representation of the result of passing <var>s</var>
       through a cryptographic hash function.
       <a>RDFC-1.0</a> uses the SHA-256 hash algorithm.</p>
+
+    <p class="note">The "degree" terminology is used within this specification
+      as coloquial way of describing
+      the <a href="https://en.wikipedia.org/wiki/Distance_(graph_theory)#Related_concepts">excentricity</a> or
```

Comment:
```suggestion
      the <a href="https://en.wikipedia.org/wiki/Distance_(graph_theory)#Related_concepts">eccentricity</a> or
```

(This is the spelling used in that Wikipedia article...)